drift_dev/lib/src/writer/manager/table_manager_writer.dart

```diff
@@ -93,6 +85,29 @@ class _TableManagerWriter {
       return true;
     }).toList();
 
+    // Remove any relations where the type isnt exactly the same (num and int)
+    // This is caused by using different type converters
```

Comment:
Also mention why we're doing this (we'd generate invalid code because we're comparing loaded fields from data classes in the manager API, right?)